src/app/components/producers/producers.component.html

```diff
@@ -30,8 +30,8 @@ <h1 class="page-header">{{ 'Producers' | translate }}</h1>
 								<tbody>
 								<tr *ngFor="let producer of producers">
 									<td><a routerLink="/accounts/{{producer.name}}">{{producer.name}}</a></td>
-									<td>{{producer.eos_balance}}</td>
-									<td>{{producer.count | number}}</td>
+									<td align="right">{{producer.eos_balance}}</td>
+                  <td><a routerLink="/blocks/{{producer.count}}">{{producer.count}}</a></td>
```

Comment:
that's an EOS balance not a block number

Reply:
Yes. EOS balance is aligned right in a table row..
producer.count is a hyperlink so you can click on it